packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReviewQuery.js

```diff
@@ -0,0 +1,40 @@
+import gql from 'graphql-tag';
+
+const ProductListingFragment = gql`
```

Comment:
This fragment should live in a file named `itemsReviewFragments.gql.js` and the fragment itself should be named `ItemsReviewFragment`.